src/components/Chart.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const Chart = () => <div>"Chart Component"</div>;
```

Comment:
@KKrzelowska this is functional React component.

No action needed.

Reply:
Maybe instead of:
`<div>"Chart Component"</div>`
You meant:
`<div>{"Chart Component"}</div>`?

The first one will probably be taken literally with apostrphes.

Reply:
I can remove apostrophes. It's only a test if the component works at all.